react/src/containers/App.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import BrewpubsIndexContainer from './BrewpubsIndexContainer';
+import BrewpubsShowContainer from './BrewpubsShowContainer';
+
+class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+
+    }
+  }
+
+  render() {
+    return(
+      <BrowserRouter>
+        <Switch>
+          <Route exact path="/brewpubs" component={BrewpubsIndexContainer} />
+          <Route exact path="/brewpubs/:id" component={BrewpubsShowContainer} />
+        </Switch>
+     </BrowserRouter>
+    )
+  }
+}
+
+export default App;
```

Comment:
I would recommend setting up your routes in a file like this instead of creating a class component because you don't need to worry about state here: https://github.com/smkopp92/manager_tracker/blob/putting-all-together/react/src/main.js